.travis.yml

```diff
@@ -21,91 +21,154 @@ install:
   - tar -xzf spark-3.0.0-bin-hadoop2.7.tgz
   - wget https://archive.apache.org/dist/spark/spark-2.4.0/spark-2.4.0-bin-hadoop2.7.tgz
   - tar -xzf spark-2.4.0-bin-hadoop2.7.tgz
+  - wget https://archive.apache.org/dist/spark/spark-3.0.0/spark-3.0.0-bin-hadoop3.2.tgz
+  - tar -xzf spark-3.0.0-bin-hadoop3.2.tgz
   - wget https://archive.apache.org/dist/hadoop/core/hadoop-2.7.7/hadoop-2.7.7.tar.gz
   - tar -xzf hadoop-2.7.7.tar.gz
+  - wget https://archive.apache.org/dist/hadoop/core/hadoop-3.2.1/hadoop-3.2.1.tar.gz
+  - tar -xzf hadoop-3.2.1.tar.gz
   - cd ${hibench}
   - cp ./travis/spark-env.sh /opt/spark-3.0.0-bin-hadoop2.7/conf/
   - cp ./travis/spark-env.sh /opt/spark-2.4.0-bin-hadoop2.7/conf/
+  - cp ./travis/spark-env.sh /opt/spark-3.0.0-bin-hadoop3.2/conf/
   - cp ./travis/core-site.xml /opt/hadoop-2.7.7/etc/hadoop/
   - cp ./travis/hdfs-site.xml /opt/hadoop-2.7.7/etc/hadoop/
   - cp ./travis/mapred-site.xml /opt/hadoop-2.7.7/etc/hadoop/
   - cp ./travis/yarn-site.xml /opt/hadoop-2.7.7/etc/hadoop/
   - cp ./travis/hadoop-env.sh /opt/hadoop-2.7.7/etc/hadoop/
-  - cp ./travis/hibench.conf ./conf/
-  - cp ./travis/hadoop.conf ./conf/
+  - cp ./travis/core-site.xml /opt/hadoop-3.2.1/etc/hadoop/
+  - cp ./travis/hdfs-site.xml /opt/hadoop-3.2.1/etc/hadoop/
+  - cp ./travis/mapred-site.xml /opt/hadoop-3.2.1/etc/hadoop/
+  - cp ./travis/yarn-site.xml /opt/hadoop-3.2.1/etc/hadoop/
+  - cp ./travis/hadoop-env.sh /opt/hadoop-3.2.1/etc/hadoop/
 jobs:
   include:
-    - name: build-multi-version
+<<<<<<< HEAD
+    - name: hibench-build-multi-version
```

Comment:
You may need to resolve the conflicts here, I rearranged the CI scripts

Reply:
@gczsjdy  Yes , I noticed it.  Thanks.